inc/auth.class.php

```diff
@@ -127,7 +127,8 @@ static function getMenuContent() {
 
 
    /**
-    * Is the user exists in the DB
+    * Check user existence in DB
+    * @global DBmysql $DB
```

Comment:
Not sure it should be used; phpdocumentor 2 will remove this tag (don't know if apigen knows about it).